data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+	name "Investigate <planet>"
+	description "Transport the Heliarch trio to <destination> so that they can investigate signs of criminal activity."
+	to offer
+		has "Heliarch Investigation 1: done"
+	source "Ring of Friendship"
+	destination "Into White"
+	to fail
+		or
+			has "Heliarch Investigation 2: declined"
+			has "Heliarch Investigation 2: aborted"
+			has "Heliarch Investigation 2 :failed"
+	on complete
+		conversation
+			`Kirrie, already donning his winter coat, instructs you to land near the equator. He says him and Tugroob will conduct some subtle interrogations, stating that it's 'all they can do' in this climate.`
+			`	You drop them off in one of the larger settlements there. After they leave, Iplora asks that you bring her close to the southern pole of the planet.`
+			`	"Suspiciously similar, independent reports on that area have been. Tasked with investigating it, I was."`
+			`	As you land on an isolated icy patch in the middle of a snowstorm, Iplora finishes her preparations, carrying a bag with some equipment and wearing what is - at least when compared to what her colleagues were wearing - fairly light clothing. Although she is a Saryd, you still worry that she might be frostbitten in this extreme weather, but she reassures you that she will be fine.`
+			`	You open your ship's hatch and she prepares to leave.`
+			choice
+				`	"Good luck. Be careful not to get lost in all this snow."`
+					goto warm
+				`	"Hold up, I'll go with you so we can search faster."`
+			`	She pauses on the ramp, looking back at you.`
+			`	"Certain of that, are you, Captain? Treacherous, these storms can be."`
+			choice
+				`	"Don't worry, I can carry my weight through some snow."`
+					goto snow
+				`	"On second thought, you're right. Good luck out there."`
+			label warm
+			`	You slouch around your ship for a few hours, waiting for any signal from the Heliarch agents, but nothing comes up on your commlink. Iplora comes back, saying she couldn't find anything, so you leave and pick up Kirrie and Tugroob. They also return empty handed.`
+				accept
+			label snow
+			`	She remains silent, and walks back inside. You close the hatch and prepare yourself for what's to come with the warmest clothing you could find. You open the hatch once more, and the both of you head out into the white fields. Not long after leaving your ship, you can make out small, dark lines nearby, probably pine trees. It seems you landed close to a forest. Unfortunately, they don't do much to protect you from the snowfall.`
+			`	You continue deeper into the lifeless forest for what feels like hours, so long that your feet start feeling numb from the cold. After yet another turn, Iplora guides you out of the forest to snowy hills. The storm is not so terrible here, so you can faintly see cave entrances in some of the more distant hills. Looking back at the woods, you see that there's a small shack in the outskirts, not too far from where you are now.`
```

Comment:
Cool, your clothing is warm. Is the outer layer waterproof? Cause if not it literally doesn't matter how warm your clothing is in this scenario.
Also, if you have proper footwear and attire, but not highly specialised footwear/attire this thing with the numb feet is about -20 centigrade, then the pain starts setting in around -26.

Reply:
Do I really need to mention that the player's winter clothing is waterproof? I thought it'd be pretty easy to infer they're not putting on a wooly sweater. Just seems like it'd be unnecessary.

For the numb feet thing, I personally never got it, but when we were on a place with snow for the first time, my father would complain about his feet going numb/hurting a bit. I imagine there's degrees to how numb your feet can feel due to cold weather, didn't mean to imply the harshest scenario with it.